### PR TITLE
A couple of changes

### PR DESCRIPTION
**Pull request by nco2k**
### NEW MODS

[Nemo's Woodcutter](https://discord.com/channels/696411537878810705/1154502772172005397) by NemoNotFound
[Nether Chest](https://github.com/Kir-Antipov/nether-chest) by Kir_Antipov
[Illager Invasion](https://github.com/Fuzss/illagerinvasion) by Fuzs
[Better Archeology](https://github.com/Pandarix/BetterArcheology) by Pandarix
[Even Better Archeology](https://github.com/noah1510/EvenBetterArcheology) by noah1510

### BUG FIXES

[TrashSlot](https://github.com/TwelveIterationMods/TrashSlot) by BlayTheNinth
- Added missing image (assets/trashslot/textures/item/trashcan.png)
- Fixed border issues (assets/trashslot/textures/gui/slot.png)

[Inventory Management](https://github.com/Roundaround/mc-fabric-inventory-management) by Roundaround
- Changed highlight color from blue to green, to match other highlights (assets/inventorymanagement/textures/gui/sprites/sort_highlighted.png)
- Changed highlight color from blue to green, to match other highlights (assets/inventorymanagement/textures/gui/sprites/stack_from_highlighted.png)
- Changed highlight color from blue to green, to match other highlights (assets/inventorymanagement/textures/gui/sprites/stack_to_highlighted.png)
- Changed highlight color from blue to green, to match other highlights (assets/inventorymanagement/textures/gui/sprites/transfer_from_highlighted.png)
- Changed highlight color from blue to green, to match other highlights (assets/inventorymanagement/textures/gui/sprites/transfer_to_highlighted.png)

Minecraft (Vanilla) Brewing Stand
- Changed item outline color to a slightly darker gray, to match other item outlines (assets/minecraft/textures/gui/container/brewing_stand.png)

### MISC

README.md
- Changed number of supported mods from 110 to 116
- Added missing credits and sorted them alphabetically

MODS-CREDITS.json
- Added missing credits and sorted them alphabetically

CONTRIBUTORS-CREDITS.json
- Added missing credits and sorted them alphabetically

### CODE REVIEW

**Comment by GabrielDja** [comment]
Thank you for your contribution,
Wow, I didn't expect so many changes.

I'll analyze your changes,
Once your changes are analyzed and accepted, this request will be closed (Because I used to do everything manually and not use the "Merge" feature of Github).

File: assets/evenbetterarcheology/lang/en_us.json
Comment (GabrielDja) [inline assets/evenbetterarcheology/lang/en_us.json:1]: The translation key is not correct:
`betterarcheology` > `evenbetterarcheology`

Comment (nco2k) [reply assets/evenbetterarcheology/lang/en_us.json:1]: oops, thanks for noticing. copy paste errors are the worst. i changed it now!

**Comment by GabrielDja** [comment]
Your changes have been applied.
Except for the "blue" to "green" color change for the Inventory Management mod, to which I had already made changes.

**Thanks for your contribution :D**

**Comment by nco2k** [comment]
cool thanks! small oversight though, you missed adding "assets/nemos-woodcutter/lang/en_us.json".

**Comment by GabrielDja** [comment]
I'll add it in the next update, thanks :D

**Pull request closed without merging**